oar/cli/cmd_image_consistency_check.py

```diff
@@ -84,8 +86,13 @@ def image_consistency_check(ctx, build_number, for_nightly):
                 "image-consistency-check", build_number)
 
             if job_status == JENKINS_JOB_STATUS_SUCCESS:
-                report.update_task_status(
-                    LABEL_TASK_IMAGE_CONSISTENCY_TEST, TASK_STATUS_PASS)
+                # TODO run the advisories check only when consistency check passed?
```

Comment:
@rioliu-rh Is this logic correct:
1. If consistency check is OK then run the checks for advisory health
2. Mark the consistency step as PASSED if advisory health is OK
3. Mark the consistency step as FAILED if advisory health is NOT OK

Reply:
pass: jenkins job is pass and advisory grade is healthy
fail: any of them is failed